database/ioc/src/Makefile

```diff
@@ -1,4 +1,4 @@
-TOP=../..
+TOP=../../..
```

Comment:
That may have been the intent, but it doesn't build w/ that TOP.

Both pvExampleTestCPP/database/src
and  pvExampleTestCPP/database/ioc/src
depend on pvExampleTestCPP/powerSupply so database
can't build standalone w/o looking up a level.

- Bruce

On 08/16/2016 12:59 PM, Andrew Johnson wrote:
>
> In database/ioc/src/Makefile <https://github.com/epics-base/pvExampleTestCPP/pull/1#discussion_r75008332>:
>
> > @@ -1,4 +1,4 @@
> > -TOP=../..
> > +TOP=../../..
>
> This change is wrong, the |pvExampleTestCPP/database| directory is an embedded top which has its own |configure| 
> directory, so from |database/ioc/src| the correct value for |TOP| is |../..|. I haven't tried this so there may need 
> to be other changes made for the build to work, but the idea here is that the |exampleDatabase| executable should be 
> installed into the |pvExampleTestCPP/database/bin/<arch>| directory, not into |pvExampleTestCPP/bin/<arch>|. This 
> technique of embedding tops is used more heavily in the newer version of the exampleCPP module that will come with the 
> V4.6.0 release.
>